src/PhpArrayGenerator.php

```diff
@@ -163,4 +172,36 @@ private static function var_export( $value ) {
 
 		return '[' . implode( ',', $entries ) . ']';
 	}
+
+	/**
+	 * Outputs or returns a parsable string representation of a variable.
+	 * @since 4.0.0
+	 *
+	 * @param mixed $value The variable you want to export.
+	 * @param int   $level The current indentation level.
+	 * @param int   $indentation The number of spaces for indentation. Default is 4.
```

Comment:
Can we do tabs instead?

Reply:
I like using tabs for my own projects, but here as it is something which will be used on other developers IDE, I prefer to use spaces, because tabs configuration depends on the configuration of user. A space will be every time a space, but tab not. Some resources not related to this subject but still on the same issue:

https://medium.com/@peey/what-is-the-inconsistent-use-of-tabs-and-spaces-in-indentation-error-and-why-is-it-caused-f3bbb8b2568b
https://stackoverflow.com/questions/5685406/inconsistent-use-of-tabs-and-spaces-in-indentation

Reply:
> I prefer to use spaces, because tabs configuration depends on the configuration of user

That's actually an advantage of tabs :-) But I don't wanna start yet another tabs vs spaces discussion here.

I am mainly suggesting using tabs because that's the WordPress coding standard and IMHO we should be consistent with that.

Reply:
Great, in this case I will try to change it to tabs to respect the standards, thanks!